ground/src/main/res/values/styles.xml

```diff
@@ -62,6 +62,13 @@
     <item name="colorPrimaryInverse">@color/md_theme_inversePrimary</item>
   </style>
 
+  <style name="AppTitle">
+    <item name="android:textColor">?attr/colorOnPrimary</item>
```

Comment:
We should use `@android:color/white` as the text color here.

Reply:
Done